cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -11,6 +11,21 @@ function cleanup() {
   virsh undefine "${DOMAIN_NAME}" || true
 }
 
+function wait_for_install_to_complete() {
+  count=0
+  retries=100
```

Comment:
I would change this function to get the total timeout and waiting interval  as arguments 
in a way we could set the timeout outside this function.

```
VIRT_INSTALL_TIMEOUT="$((60*8))"
...
function wait_for_install_to_complete() {
    local -r timeout_seconds=$1
    local -r interval_seconds=$2
    local -r count=0
    local -r retries=$((timeout_seconds\interval_seconds))
...
}
...
wait_for_install_to_complete "${VIRT_INSTALL_TIMEOUT}" "5"

It will also make it easier to tune it on CI directly from the prow-job yaml.